packages/keyring/src/testing.ts

```diff
@@ -73,6 +73,49 @@ export const PAIRSSR25519: PairDef[] = [
 ];
 
 export const PAIRSETHEREUM: PairDef[] = [
+  {
+    name: 'Alith',
+    publicKey: hexToU8a('0x02509540919faacf9ab52146c9aa40db68172d83777250b28e4679176e49ccdd9f'),
+    secretKey: hexToU8a('0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133'),
+    seed: 'Alith',
```

Comment:
No sure the seed it correct? I'm assuming it is something like `m/44'/40'/0'/0`?

Reply:
(so in the case of the sr25519 is is basically the `//Alice` with the `//` removed (not quire correct removing it, but it made some other parts simpler, so it "sort-of stuck")

Reply:
yeah no the seed is not correct. I added it for typing's sake. I guess I should add the seed derived from the mnemonic? Didn't think it would matter.

Reply:
`m/44'/40'/0'/0` is not the seed, it's the derivation path used to derive the privatekey from the seed

Reply:
The "seed" here is used intechagably because of historic reasons -

- in the original version the "seed" was used as-is to derive keys
- in later versions it changed to `<known mnemonic>//<original seed>` (name was not changed)

So it does refer to the path.

Reply:
good to know thanks. updated

Reply:
It should be changed and the actual sr* should really reflect the full path. Bug me each time I look at this file, but not enough to fix yet since it is "cosmetic". (Additionally the old seed-like derivation can be removed from the body)